qiskit/transpiler/passes/calibration/builders.py

```diff
@@ -103,26 +103,32 @@ class RZXCalibrationBuilder(CalibrationBuilder):
     angle. Additional details can be found in https://arxiv.org/abs/2012.11660.
     """
 
-    def __init__(self, backend: basebackend):
```

Comment:
The `from qiskit.providers import basebackend` import up at the top will be unused now.